www/source/partials/tutorials/_web_guide_connect_database.slim

```diff
@@ -314,13 +314,13 @@ section
 
    = code(:shell) do
      | 
-      $ sudo hab start core/postgresql --topology leader --group production
+      $ sudo hab sup run core/postgresql --topology leader --group production
```

Comment:
This and the other instance in this file need to be fixed. See https://www.habitat.sh/blog/2017/08/habitat-rails-postgres-3-different-ways/